### DISCUSSION TITLE
Leetcode solving

### DISCUSSION THREAD

**Discussion by vlabeemir** [apple/pkl#250] [General]
Is it possible to solve LeetCode problems using only pkl?
Is it worth learning pkl as a second programming language after Python?

**Comment by bioball** [comment]
You can certainly solve LeetCode problems in Pkl. Pkl isn't a general purpose language, though. If you are looking to expand your skills as a programmer, I probably would recommend learning a compiled language next (Swift, Go, or Java are all great to learn).

Pkl is a somewhat niche language because it's specifically meant for configuration.

**Reply by vlabeemir** [reply]
Thank you
